src/legacy/server/config/schema.js

```diff
@@ -246,6 +246,9 @@ export default () =>
         faviconUrl: Joi.any().default('/'),
         applicationTitle: Joi.any().default(''),
       }),
+      multipleDataSource: Joi.object({
+        enabled: Joi.boolean().default(true),
```

Comment:
why default to true? will this conflict with the config defined in `src/core/server/opensearch_dashboards_config.ts`?

Reply:
> why default to true? will this conflict with the config defined in `src/core/server/opensearch_dashboards_config.ts`?

good catch!

Reply:
changed to false